sdk/src/main/scala/com/horizen/account/forger/AccountForger.scala

```diff
@@ -0,0 +1,59 @@
+package com.horizen.account.forger
+
+import akka.actor.{ActorRef, ActorSystem, Props}
+import com.horizen._
+import com.horizen.account.companion.SidechainAccountTransactionsCompanion
+import com.horizen.account.history.AccountHistory
+import com.horizen.account.mempool.AccountMemoryPool
+import com.horizen.account.state.AccountState
+import com.horizen.account.storage.AccountHistoryStorage
+import com.horizen.account.wallet.AccountWallet
+import com.horizen.forge.{AbstractForger, MainchainSynchronizer}
+import com.horizen.params.NetworkParams
+
+import scorex.core.utils.NetworkTimeProvider
+
+
+class AccountForger(settings: SidechainSettings,
+             viewHolderRef: ActorRef,
+             forgeMessageBuilder: AccountForgeMessageBuilder,
+             timeProvider: NetworkTimeProvider,
+             params: NetworkParams) extends AbstractForger(
```

Comment:
`extends AbstractForger[SidechainTypes#SCAT, AccountBlockHeader, AccountBlock](...`
similar for `Forger`

Reply:
Done